HARK/cAndCwithStickyE/StickyEtools.py

```diff
@@ -532,76 +536,76 @@ def makeResultsPanel(Coeffs,StdErrs,Rsq,Pvals,OID,Counts,meas_err,sticky,all_spe
     '''
     # Define Delta log C text and expectations text
     if meas_err:
-        DeltaLogC = '$\Delta \log \mathbf{C}_{t}^*$'
-        DeltaLogC1 = '$\Delta \log \mathbf{C}_{t+1}^*$'
-        MeasErr = ' (with measurement error $\mathbf{C}_{t}^* =\mathbf{C}_{t}\\times \\xi_t$);'
+        DeltaLogC = r'$\Delta \log \mathbf{C}_{t}^*$'
+        DeltaLogC1 = r'$\Delta \log \mathbf{C}_{t+1}^*$'
+        MeasErr = r' (with measurement error $\mathbf{C}_{t}^* =\mathbf{C}_{t}\\times \\xi_t$);'
     else:
-        DeltaLogC = '$\Delta \log \mathbf{C}_{t}$'
-        DeltaLogC1 = '$\Delta \log \mathbf{C}_{t+1}$'
-        MeasErr = ' (no measurement error)'
+        DeltaLogC = r'$\Delta \log \mathbf{C}_{t}$'
+        DeltaLogC1 = r'$\Delta \log \mathbf{C}_{t+1}$'
+        MeasErr = r' (no measurement error)'
 
     if sticky:
-        Expectations = 'Sticky'
+        Expectations = r'Sticky'
     else:
-        Expectations = 'Frictionless'
+        Expectations = r'Frictionless'
 
-    DeltaLogY1 = '$\Delta \log \mathbf{Y}_{t+1}$'
-    A_t = '$A_{t}$'
+    DeltaLogY1 = r'$\Delta \log \mathbf{Y}_{t+1}$'
+    A_t = r'$A_{t}$'
 
 
     # Define significance symbols
     if Counts[2] > 1:
-        sig_symb = '\\bullet '
+        sig_symb = r'\\bullet '
     else:
-        sig_symb = '*'
+        sig_symb = r'*'
     def sigFunc(coeff,stderr):
         z_stat = np.abs(coeff/stderr)
         cuts = np.array([1.645,1.96,2.576])
         N = np.sum(z_stat > cuts)
         if N > 0:
-            sig_text = '^{' + N*sig_symb + '}'
+            sig_text = r'^{' + N*sig_symb + '}'
         else:
-            sig_text = ''
+            sig_text = r''
         return sig_text
 
     # Make the memo text
-    memo = ''
+    memo = r''
     if all_specs:
-        memo += '\\\\ \multicolumn{6}{l}{Memo: For instruments $\mathbf{Z}_{t}$, ' + DeltaLogC + ' $= \mathbf{Z}_{t} \zeta, ~\\bar{R}^{2}=$ ' + mystr1(Counts[3])
+        memo += r'\\\\ \multicolumn{6}{l}{Memo: For instruments $\mathbf{Z}_{t}$, ' + DeltaLogC + ' $= \mathbf{Z}_{t} \zeta, ~\\bar{R}^{2}=$ ' + mystr1(Counts[3])
         if meas_err:
-            memo += '; ~$\\var(\\log(\\xi_t))=$ ' + mystr3(Counts[4])
-        memo += ' }  \n'
+            memo += r'; ~$\\var(\\log(\\xi_t))=$ ' + mystr3(Counts[4])
+        memo += r' }  \n'
```

Comment:
Same raw string literal concern here: this newline character just turned into a backslash followed by the letter `n`. Is that what you wanted?